.github/workflows/docs.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - uses: josh146/sphinx-action@master
+    - uses: XanaduAI/sphinx-action@master

```

Comment:
We should also this opportunity to remove the `repo-token` field!

Reply:
What was the `repo-token` used for and why is it no longer needed? Can we just remove that line (leaving everything else as it is)?